Jenkinsfile-jdk11

```diff
@@ -72,14 +72,15 @@ pipeline {
         stage('Build') {
             steps {
                 echo 'Building'
-                sh 'mvn -U -B -e clean install -DskipTests apache-rat:check'
+                sh 'mvn -U -B -e apache-rat:check'
```

Comment:
A bit of a cheat, execute "rat" using the global repo (shared with all builds) `~/.m2/repository`
Other maven executions use `.repository` in the clean checkout directory.

Alternatively, I can configure the rat plugin to ignore `.repository` and add it to .gitignore?

Reply:
Sigh. That's why maven has a per-workspace repo. That shouldn't be needed and became a little better through the matrix build on main.

Or just do the rat check on jdk11 only.

Reply:
+1 for this, but I'm going to move it in its own stage for now, and try to remove the duplication between these files on `main`.

Hopefully something like this in each Jenkins file:
```groovy
const jdkVersion = 'jdk_8_latest'
const deploy = false
const rat = false
const supportedBranch = (env.BRANCH_NAME == "1.7.x" || env.BRANCH_NAME == "1.8.x" || env.BRANCH_NAME == "main") // we have a lot of this in the individual files

load('Jenkinsfile-common')
}
```

I'm not sure that will work (given the scope of those globals 🤮), but I'll poke around with it

Reply:
oh, this is already resolved on `main` with a matrix build 😄